### ISSUE TITLE
Video dataset finetuning problem

### ISSUE THREAD

**Issue by Afssal** [huggingface/smollm#76]
I tried fine-tuning the SmolVLM model with 500M parameters, and the training and validation losses are as follows:
Step | Train Loss | Val Loss
50 | 0.005800 | 0.010622
100 | 0.005600 | 0.010517
150 | 0.005200 | 0.012322
200 | 0.007700 | 0.010791
250 | 0.005300 | 0.010654
300 | 0.006200 | 0.010917
350 | 0.005800 | 0.010357
400 | 0.006000 | 0.011858
450 | 0.005300 | 0.010400
500 | 0.006600 | 0.011113
550 | 0.005700 | 0.010420
600 | 0.005900 | 0.010824
650 | 0.005300 | 0.010427
700 | 0.005200 | 0.010276
750 | 0.005400 | 0.010645
800 | 0.006400 | 0.010399
850 | 0.005600 | 0.010369
900 | 0.006900 | 0.010382
950 | 0.005200 | 0.010427
1000 | 0.005300 | 0.010427

However, when I tried inference, I got the following output:
User: Caption the video. You are provided the following series of three frames from a 0:00:03 [H:MM:SS] video.
Frame from 00:00:
Frame from 00:01: [00:01:01] [00:01:01] [00:01:01]
Frame from 00:01:02: [00:01:02:01] [00:01:

What could be the issue here?

**Comment by Jing-Fu** [comment]
What version of transformers do you have in your environment?

**Comment by Jing-Fu** [comment]
Maybe you can refer to my approach. 
[https://github.com/huggingface/smollm/issues/75](url)

**Comment by Afssal** [comment]
> What version of transformers do you have in your environment?
I installed transformers from source using command pip install -q git+https://github.com/huggingface/transformers.git

**Comment by Afssal** [comment]
> Maybe you can refer to my approach. [#75](https://github.com/huggingface/smollm/issues/75)

It's Working ! Thank you
